tribits/examples/TribitsExampleProject/packages/with_subpackages/a/A.hpp

```diff
@@ -11,6 +11,9 @@ namespace WithSubpackages {
   // return a string describing the dependencies of "A", recursively
   std::string depsA();
 
+  // return speical value
```

Comment:
`special`

Reply:
Resolved in new commit https://github.com/TriBITSPub/TriBITS/pull/520/commits/b26f5ac73332cc247e7d94b19431663dd761077d